### ISSUE TITLE
Nested role JSON responses cannot be configured

### ISSUE THREAD

**Issue by flo-mic** [9p4/jellyfin-plugin-sso#8]
I try to setup an managed access control from keycloak. Unfortunately I'm not able to do so and need some help. The SSO connection itself is working and I can access jellyfin with a keycloak user. But I'm not able to create a user with admin privileges at the moment.

Within my access token I have the following structure for the roles
```
"resource_access":{"jellyfin-auth-client":{"roles":["jellyfin-admin"]}},"scope":"openid profile email"
```

The oid api config looks like this, as you can see I define the role claim and admin roles to the matching results in the access token. But somehow the users are not created with admin privileges and in the logs I can see that the users are assigned to an InvalidAuthProvider.
```
{
    ...
    
    "Enabled": true,
    "EnableAuthorization": true,
    "EnableAllFolders": true,
    "EnabledFolders": [],
    "AdminRoles": [
        "jellyfin-admin"
    ],
    "Roles": [],
    "EnableFolderRoles": false,
    "FolderRoleMapping": [],
    "RoleClaim": "resource_access.jellyfin-auth-client"
}
```

Console log:
```
[13:49:17] [INF] [36] Jellyfin.Plugin.SSO_Auth.Api.SSOController: SSO Controller initialized
[13:49:17] [INF] [36] Jellyfin.Plugin.SSO_Auth.Api.SSOController: SSO user doesn't exist, creating...
[13:49:18] [INF] [36] Jellyfin.Plugin.SSO_Auth.Api.SSOController: Auth request created...
[13:49:18] [INF] [36] Emby.Server.Implementations.Session.SessionManager: Current/Max sessions for user user5: 0/0
[13:49:18] [INF] [36] Emby.Server.Implementations.Session.SessionManager: Creating new access token for user d5772d84-f5d6-4f97-8bd6-0a85e6eba4f9
[13:49:18] [WRN] [36] Jellyfin.Server.Implementations.Users.UserManager: User user5 was found with invalid/missing Authentication Provider Jellyfin.Plugin.SSO_Auth.Api.SSOController. Assigning user to InvalidAuthProvider until this is corrected
[13:49:18] [WRN] [37] Jellyfin.Server.Implementations.Users.UserManager: User user5 was found with invalid/missing Authentication Provider Jellyfin.Plugin.SSO_Auth.Api.SSOController. Assigning user to InvalidAuthProvider until this is corrected
[13:49:18] [WRN] [3] Jellyfin.Server.Implementations.Users.UserManager: User user5 was found with invalid/missing Authentication Provider Jellyfin.Plugin.SSO_Auth.Api.SSOController. Assigning user to InvalidAuthProvider until this is corrected
[13:49:22] [WRN] [21] Jellyfin.Server.Implementations.Users.UserManager: User user5 was found with invalid/missing Authentication Provider Jellyfin.Plugin.SSO_Auth.Api.SSOController. Assigning user to InvalidAuthProvider until this is corrected
```

**Comment by 9p4** [comment]
Yeah, nested role attributes aren't implemented yet. I'll get working on that.

**Comment by 9p4** [comment]
Should be fixed in #1a736d9707893bb3fe0ca7ac1ebcf02f2b6fe5d6
